pandas/plotting/_core.py

```diff
@@ -468,12 +468,13 @@ def _adorn_subplots(self):
                 self.axes[0].set_title(self.title)
 
     def _apply_axis_properties(self, axis, rot=None, fontsize=None):
-        labels = axis.get_majorticklabels() + axis.get_minorticklabels()
-        for label in labels:
-            if rot is not None:
-                label.set_rotation(rot)
-            if fontsize is not None:
-                label.set_fontsize(fontsize)
+        if rot is not None or fontsize is not None:
```

Comment:
```suggestion
        if rot or fontsize:
```

Reply:
This is quite different - `rot = 0` is valid and common (`fontsize = 0` makes less sense, but seems to be valid to `matplotlib`).

Given that `rot=0` is the default, your suggestion might be okay but I can't guarantee it works in all cases.

Reply:
Makes sense - was only thinking about None / False didn't consider 0